recipe/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sarif_om" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: cd5f416b3083e00d402a92e449a7ff67af46f11241073eea0461802a3b5aef98
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
+    - setuptools
+    - pbr ==5.6.0
+  run:
+    - python
+    - pbr
```

Comment:
You can technically remove this. It depends on pip and we don't really want pip to be in our dependencies. `pbr` is a build time dep.

Reply:
I did throw `pbr` out of run and it was getting errors, putting it back in the below commit fixed it. Any thoughts what to do about that? 

https://github.com/AnacondaRecipes/sarif-python-om-feedstock/pull/1/commits/33c7afde471eded730fd23eca19b14caf8fe4986

Reply:
Patching setup.cfg to remove pbr from install_requires would solve your test issue.

Reply:
Cheers good should will do that now